### PR TITLE
Newsletter for Jan 29th

### PR DESCRIPTION
**Pull request by tomgenoni**

### CODE REVIEW

File: www/src/pages/updates/notes/2019-01-29.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: January 29, 2019
+---
+
+## 🎉 Thumbprint relaunch
```
Comment (danoc) [inline www/src/pages/updates/notes/2019-01-29.mdx]:
```suggestion
## 🎉 Thumbprint public launch
```

A little more clear than relaunch IMO.

File: www/src/pages/updates/notes/2019-01-29.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Today we officially launched a newly redesigned [Thumbprint](https://thumbprint.design). Content has been reorganized and section links are always available from the trusty sidebar.
+
+We also published an [introduction](https://medium.com/thumbtack-design/an-introduction-to-thumbprint-61f1dfde5c93) on our Thumbtack Design account on Medium.
```
Comment (danoc) [inline www/src/pages/updates/notes/2019-01-29.mdx]:
```suggestion
We also published an [introduction to Thumbprint](https://medium.com/thumbtack-design/an-introduction-to-thumbprint-61f1dfde5c93) on our Thumbtack Design account on Medium.
```

File: www/src/pages/updates/notes/2019-01-29.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: January 29, 2019
+---
+
+## 🎉 Thumbprint relaunch
+
+Today we officially launched a newly redesigned [Thumbprint](https://thumbprint.design). Content has been reorganized and section links are always available from the trusty sidebar.
+
+We also published an [introduction](https://medium.com/thumbtack-design/an-introduction-to-thumbprint-61f1dfde5c93) on our Thumbtack Design account on Medium. Feel free to share.
```
Comment (danoc) [inline www/src/pages/updates/notes/2019-01-29.mdx]:
```suggestion
We also published an [introduction to Thumbprint](https://medium.com/thumbtack-design/an-introduction-to-thumbprint-61f1dfde5c93) on our Thumbtack Design account on Medium. Feel free to share.
```

File: www/src/pages/updates/notes/2019-01-29.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## 💾 Now open source
+
+Thumbprint is now an officially open source project and our [repository](https://github.com/thumbtack/thumbprint) is public. This brings with it some advantages:
+
+-   Gives back to wider open source community that served as inspiration for us.
+-   Encourage contributions to Thumbprint.
+-   Provides prospective employees a view into our process and technology.
+
+Huge credit goes to Dan C. for leading this effort, moving around a ton of code, and keeping it all on track.
```
Comment (danoc) [inline www/src/pages/updates/notes/2019-01-29.mdx]:
```suggestion
Huge credit goes to Dan O. for leading this effort, moving around a ton of code, and keeping it all on track.
```

lolol

File: www/src/pages/updates/notes/2019-01-29.mdx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Special thanks to Jon K. for help with the redesign and QA.
+
+## 💾 Now open source
+
+Thumbprint is now an officially open source project and our [repository](https://github.com/thumbtack/thumbprint) is public. This brings with it some advantages:
+
+-   Gives back to wider open source community that served as inspiration for us.
+-   Encourage contributions to Thumbprint.
+-   Provides prospective employees a view into our process and technology.
```
Comment (danoc) [inline www/src/pages/updates/notes/2019-01-29.mdx]: Thoughts on putting this first and making it more direct?

> Assist with design and engineering recruiting by providing prospective employees a view into our process and technology

Or:

> Assist with design and engineering recruiting by promoting the Thumbtack brand

Comment (tomgenoni) [reply www/src/pages/updates/notes/2019-01-29.mdx]: The first one sounds better to me, more concrete.

**Pull request merged**